usr.sbin/cron/cron/popen.c

```diff
@@ -74,7 +74,7 @@ cron_popen(program, type, e, pidptr)
 	char **pop, *vv[2];
 	int gargc;
 	char *gargv[1000];
-	extern char **glob(), **copyblk();
+	extern char **glob(void), **copyblk(void);
```

Comment:
These are also wrong. These take args.
I'm unsure where it's snagging these from. glob looks to come from libc, not sure copyblk.
glob's .h file is glob.h.